src/test/java/org/folio/processing/mapping/InstanceMappingTest.java

```diff
@@ -461,6 +471,84 @@ public void testMarcToInstanceWithRepeatableISBN() throws IOException {
     });
   }
 
+  @Test
+  public void testMarcToInstanceWithRepeatableSubects() throws IOException {
+    final String FIRST_LIBRARY_SOURCE_ID = "e894d0dc-621d-4b1d-98f6-6f7120eb0d40";
+    final String SECOND_LIBRARY_SOURCE_ID = "e894d0dc-621d-4b1d-98f6-6f7120eb0d40";
```

Comment:
```suggestion
```
looks like this variable contains the same UUID as the previous one and is never used in the test

Reply:
@RuslanLavrov  Thank you! I changed this logic.